test/extensions/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include "mavis/extension_managers/RISCVExtensionManager.hpp"
+
+int main(int argc, char* argv[])
```

Comment:
Do you have test for an invalid ISA string? For example, if you ISA spec says that "d" requires "f", but your ISA string is `rv64imad`.

Reply:
Good call, I've added some failing test cases.